src/bindings/js/node/src/node_output.cpp

```diff
@@ -45,7 +45,14 @@ Napi::Value Output<ov::Node>::get_partial_shape(const Napi::CallbackInfo& info)
 }
 
 Napi::Value Output<ov::Node>::get_any_name(const Napi::CallbackInfo& info) {
-    return Napi::String::New(info.Env(), _output.get_any_name());
+    std::string name;
+    if (_output.get_names().empty()) {
+        name = _output.get_node()->get_name();
+    } else {
+        name = _output.get_any_name();
+    }
```

Comment:
```suggestion
    const std::string& js_model_output_name(output){
        return output.get_names().empty() ? output.get_node()->get_name() : output.get_any_name()
    }
```

Suggest to make js internal helper to get this name. If there will be change it will be easier to replace or update it.
Also tak into consideration that will work only for OV output which belongs node with single output. for Nodes which have got many outputs the same tensor name will be generated.